test_communication/test/test_secure_publisher_subscriber.py.in

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
Should have a comment line to mention which template this file has been generated from.